pkg/etcd/etcd.go

```diff
@@ -65,7 +71,19 @@ type GetterSetter interface {
 // Returns:
 // 	This puts an *etcd.Client into the context.
 func CreateClient(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
-	url := p.Get("url", "http://localhost:4001").(string)
+	host := os.Getenv(hostEnvVar)
+	port, err := strconv.Atoi(os.Getenv(portEnvVar))
+	if host == "" {
+		host = defaultHost
+	}
+	if err != nil {
+		port = 4001
+	}
+
+	// Aaron(12/9/2015): to shame me into fixing this later, but it's a bigger change
+	fmt.Println("Aaron hasn't fixed the env var collection stage in the cookoo route yet...")
+	// url := p.Get("url", "http://localhost:4001").(string)
+	url := fmt.Sprintf("%s:%d", host, port)
 
```

Comment:
@smothiki can't reproduce it now, so I'll back out this change. I ran into the issue when I was setting everything up, so could have just been a n00b mistake.